paper-server/src/main/java/org/bukkit/craftbukkit/CraftMusicInstrument.java

```diff
@@ -63,8 +65,28 @@ public CraftMusicInstrument(Holder<Instrument> holder) {
     }
 
     @Override
-    public Holder<Instrument> getHolder() { // Paper - switch to Holder
-        return this.holder; // Paper - switch to Holder
+    public Holder<Instrument> getHolder() {
+        return this.holder;
+    }
+
+    @Override
+    public float getDuration() {
+        return this.getHandle().useDuration();
+    }
+
+    @Override
+    public float getRange() {
+        return this.getHandle().range();
+    }
+
+    @Override
+    public Component description() {
```

Comment:
Just do getDescription to keep it consistent with the rest, we don't have to be scared of upstream conflicts anymore